### PR TITLE
Improved docs

### PR DESCRIPTION
**Pull request by inigomarquinez**
### Main changes

- :wrench: added nvmrc file
- :wrench: added new contributors
- :memo: improved readme
- :art: added assets for logo and banner

### Screenshots

Captura de Pantalla 2022-04-26 a las 11 59 48

Captura de Pantalla 2022-04-26 a las 11 32 16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
+  npm install @guidesmiths/license-checker
+  ```
 
 - Add a script to run the package
 
-```
-npx @guidesmiths/license-checker --failOn license1,license2
-```
+  ```
+  npx @guidesmiths/license-checker --failOn license1,license2
```
Comment (ardguezsoc) [inline README.md:32]: con la version 2.0 hay que acordarse supongo que cambiar esto en el readme no ?

**Review by ardguezsoc — APPROVED** [review]
giphy

File: README.md (hunk 1 above)
Comment (inigomarquinez) [reply README.md:32]: CIERTO!!! Lo cambio

Comment (inigomarquinez) [reply README.md:32]: Oh, I've just remembered that this has already been updated in the corresponding PR made by @neodmy

**Review by ardguezsoc — APPROVED** [review]

**Review by dustytrinkets — APPROVED** [review]

**Pull request merged**